native/packages/react-native-bpk-component-banner-alert/src/AnimateAndFade.js

```diff
@@ -16,15 +16,60 @@
  * limitations under the License.
  */
 
+/* @flow */
+
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React, { Component, type Node } from 'react';
 import { View, Animated, ViewPropTypes } from 'react-native';
 import { animationDurationSm } from 'bpk-tokens/tokens/base.react.native';
 
 const COLLAPSED_HEIGHT = 0.01;
 
-class AnimateAndFade extends Component {
-  constructor(props) {
+type MeasureCallback = (
```

Comment:
The signature of RN's measure callback -> https://facebook.github.io/react-native/docs/direct-manipulation.html#measurecallback